pkg/remotewrite/trend.go

```diff
@@ -215,3 +214,14 @@ func toBucketSpanProto(s []*dto.BucketSpan) []*prompb.BucketSpan {
 	}
 	return spans
 }
+
+func baseUnit(vt metrics.ValueType) string {
```

Comment:
What if we call it `unitSuffix` and return the actual suffix (`_seconds` and `_bytes`),  then later we don't need any condition like `if suffix != ""` but, just `v := defaultMetricPrefix + series.Metric.Name + suffix`

Reply:
I'm not getting how you would manage the case when the ValueType is Undefined (so not Time nor Data).

Reply:
just return an empty sting (a.k.a. null suffix :sweat_smile: )

Reply:
>if suffix != "" but, just v := defaultMetricPrefix + series.Metric.Name + suffix

As shared in private, this logic is applied from `MapSeries` which is a general function where the baseUnit is one of the possible options for the suffix but not the unique.